db/index_test.go

```diff
@@ -166,7 +166,7 @@ func getUsersIndexDescOnWeight() client.IndexDescription {
 	return client.IndexDescription{
 		Name: testUsersColIndexWeight,
 		Fields: []client.IndexedFieldDescription{
-			{Name: usersWeightFieldName, Direction: client.Ascending},
+			{Name: usersWeightFieldName},
```

Comment:
question: Shouldn't we still test we direction ascending?

Reply:
it's not strictly necessary, but I can add it for completeness